### ISSUE TITLE
Villager carrer id

### ISSUE THREAD

**Issue by LoToRTv** [Nathat23/StackMob-2#109]
Hello how fix error ? 
cfg plugin default 
mc 1.13.1
bungee on

>[18:53:52 WARN]: [StackMob] Task #75 for StackMob v2.4.1 generated an exception
>java.lang.IllegalArgumentException: Career id must be greater than 0
>        at org.apache.commons.lang.Validate.isTrue(Validate.java:136) ~[spigot.jar:git-Spigot-6b015b4-a4bdecf]
>        at org.bukkit.craftbukkit.v1_13_R2.entity.CraftVillager.getCareer(CraftVillager.java:134) ~[spigot.jar:git-Spigot-6b015b4-a4bdecf]
>        at org.bukkit.craftbukkit.v1_13_R2.entity.CraftVillager.getCareer(CraftVillager.java:54) ~[spigot.jar:git-Spigot-6b015b4-a4bdecf]
>        at uk.antiperson.stackmob.tools.EntityTools.notMatching(EntityTools.java:70) ~[?:?]
>        at uk.antiperson.stackmob.tools.EntityTools.notEnoughNearby(EntityTools.java:293) ~[?:?]
>        at uk.antiperson.stackmob.listeners.entity.SpawnEvent$1.run(SpawnEvent.java:94) ~[?:?]
>        at org.bukkit.craftbukkit.v1_13_R2.scheduler.CraftTask.run(CraftTask.java:76) ~[spigot.jar:git-Spigot-6b015b4-a4bdecf]
>        at org.bukkit.craftbukkit.v1_13_R2.scheduler.CraftScheduler.mainThreadHeartbeat(CraftScheduler.java:361) [spigot.jar:git-Spigot-6b015b4-a4bdecf]
>        at net.minecraft.server.v1_13_R2.MinecraftServer.b(MinecraftServer.java:893) [spigot.jar:git-Spigot-6b015b4-a4bdecf]
>        at net.minecraft.server.v1_13_R2.DedicatedServer.b(DedicatedServer.java:411) [spigot.jar:git-Spigot-6b015b4-a4bdecf]
>        at net.minecraft.server.v1_13_R2.MinecraftServer.a(MinecraftServer.java:835) [spigot.jar:git-Spigot-6b015b4-a4bdecf]
>        at net.minecraft.server.v1_13_R2.MinecraftServer.run(MinecraftServer.java:733) [spigot.jar:git-Spigot-6b015b4-a4bdecf]

**Comment by Nathat23** [comment]
What plugins do you have installed? @LoToRTv

**Comment by LoToRTv** [comment]
Sorry for waiting  https://imgur.com/a/7ZyZ3WM

**Comment by Baktus79** [comment]
I believe I found the cause. I see that myself and @LoToRTv have the plugin Citizens. I noticed today that one of my admins did create a villager NPC through Citizens. When I did remove those the spamming stopped. I did this few minutes ago and still need it to run through the day to see if that was the cause, but for me it seems like it was.

Is it possible for you to make it skip villagers made by Citizens?
